### ISSUE TITLE
A message to recruiters regarding how to judge Github contributions.

### ISSUE THREAD

**Issue by atlassubbed** [isaacs/github#1290]
I can't say judging people by their contributions is good or bad. As with all information, there are downsides and upsides. If you're going to judge people this way, you should know that **by increasing contributions, the net intensity of the chart's colors may decrease, making the chart look "worse".** In order to properly judge a user based on their contributions, you must look at values and not *only* colors.

---

Some background: There are five colors, each of which represent an "intensity" of contributions on a certain day, going from light gray to dark green. Over the past week, I've been uploading some of my projects to Github and I noticed something "strange":

My chart over the last week not counting today:
  1. Values: `[26, 50, 39, 37]`
  2. Colors: Darkest: 3/4, Second Darkest: 1/4

Today, I've made 58 contributions, so the new array is:
  1. Values: `[26, 50, 39, 37, 58]`
  2. Colors: Darkest: 2/4, Second Darkest: 2/4, Third Darkest: 1/4

By contributing *more* to Github, my chart actually started to appear *less* impressive!

**Comment by nombrekeff** [comment]
Yes, I noticed this, but didn't pay much attention, but now that you say it its not a good behaviour.
